cmd/kuiper/main.go

```diff
@@ -64,22 +63,20 @@ func main() {
 	//		Usage: "the name of stream",
 	//	}}
 
-	b, err := conf.LoadConf(clientYaml)
+	var cfg map[string]clientConf
+	err := conf.LoadConfig(&cfg)
```

Comment:
This is wrong. Here we should read clientYaml: client.yaml but the LoadConfig read from kuiper.yaml

Reply:
Fixed